Dockerfile

```diff
@@ -27,9 +27,7 @@ COPY application/ /data/
 
 # Fix folder permissions
 RUN chown -R www-data:www-data \
-    console/runtime/ \
-    frontend/runtime/ \
-    frontend/web/assets/
```

Comment:
Looks like it was a timing issue off of `develop`, you should not see that problem using this branch.